lib/kontena/machine/digital_ocean/cloudinit.yml

```diff
@@ -48,6 +52,12 @@ coreos:
         Name=weave*
 
         [Network]
+    - name: zz-default.network
+      runtime: false
+      content: |
+        # default should only match real network interfaces
+        [Match]
```

Comment:
Specifically for DO, the CoreOS `zz-default.network` unit is mostly useless, because the DO CoreOS ships with a `/usr/share/oem/cloud-config.yml` that handles the DO network interface configuration via `.network` units under `/run/systemd/network/`.